.github/workflows/NodeCI.yml

```diff
@@ -30,9 +28,9 @@ jobs:
       with:
         node-version: ${{ matrix.node-version }}
     - name: Install Packages
-      run: npm install --legacy-peer-deps
+      run: yarn install --ignore-engines
```

Comment:
@ota-meshi 

So in theory, `eslint-plugin-svelte@v2` does support node v12 well, is there any reason to enforce deprecating right now?